engine/transaction/draft/outputs/op_return.go

```diff
@@ -0,0 +1,65 @@
+package outputs
+
+import (
+	"context"
+	"encoding/hex"
+	"errors"
+
+	"github.com/bitcoin-sv/go-sdk/script"
+	sdk "github.com/bitcoin-sv/go-sdk/transaction"
+	"github.com/bitcoin-sv/spv-wallet/engine/spverrors"
+	"github.com/bitcoin-sv/spv-wallet/engine/transaction"
+	txerrors "github.com/bitcoin-sv/spv-wallet/engine/transaction/errors"
+	"github.com/bitcoin-sv/spv-wallet/models/request/opreturn"
+)
+
+// OpReturn represents an OP_RETURN output specification.
+type OpReturn opreturn.Output
+
+func (o *OpReturn) evaluate(context.Context) (annotatedOutputs, error) {
+	if len(o.Data) == 0 {
+		return nil, txerrors.ErrDraftOpReturnDataRequired
+	}
+
+	data, err := o.getData()
```

Comment:
Maybe we should iterate through this data and check if it doesn't exceed max size (80 bytes??).

Reply:
you mean to validate if it doesn't exceed max size of push data to tx?
I was thinking about it, but actually the same check is done inside sdk.CreateOpReturnOutput(data) so I'm just pushing this responsibility to sdk.
If you're asking about max size of transaction, IMHO it is rather responsibility of the "broadcasting", because there is a consensus in the network of the transaction size, and how spv-wallet can know what is the current max size. Or we will be forced to maintain this size by ourselves.

Reply:
Yes, I was thinking about max size of push data but if it's already in sdk it's ok